pkg/cmd/server/server.go

```diff
@@ -865,13 +801,6 @@ func (s *server) runControllers(defaultVolumeSnapshotLocations map[string]string
 	// by v2.0, the block from this line and including the `s.mgr.Start() will be
 	// deprecated, since the manager auto-starts all the caches. Until then, we need to start the
 	// cache for them manually.
-	for i := range controllers {
-		controllerRunInfo := controllers[i]
-		// Adding the controllers to the manager will register them as a (runtime-controller) runnable,
-		// so the manager will ensure the cache is started and ready before all controller are started
-		s.mgr.Add(managercontroller.Runnable(controllerRunInfo.controller, controllerRunInfo.numWorkers))
-	}
-
 	s.logger.Info("Server starting...")
 
 	if err := s.mgr.Start(s.ctx); err != nil {
```

Comment:
Do we still need `s.mgr.Start`? @ywk253100 @blackpiglet

Reply:
This one is needed.